filament/backend/src/opengl/OpenGLDriver.cpp

```diff
@@ -2456,6 +2506,7 @@ void OpenGLDriver::setPresentationTime(int64_t monotonic_clock_ns) {
 void OpenGLDriver::endFrame(uint32_t frameId) {
     //SYSTRACE_NAME("glFinish");
     //glFinish();
+    //executeGpuCommandsCompleteOps();
```

Comment:
Remove?

Reply:
This is an interesting way to debug certain problems -- after a finish, all pending fences should have signaled.